### PR TITLE
Use lists instead of numpy in gpt-j tokenizer and remove cell captures

### PR DESCRIPTION
**Pull request by payoto**
There is a bug in the gpt-j notebook which causes it to fail on pre-processing steps.

I also remove the %%capture which hide progress from users and make it difficult to identify changes in packages.

This was tested locally: https://graphcore.atlassian.net/browse/AIP-61

tests are still ongoing to check that  the accuracy of the model is good enough, but have gone through the initial error (it took 2mn to fail and is now 8mn through)

### CODE REVIEW

File: gptj-text-generation/data/mnli_data.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def __call__(self, items):
 
 def tokenizes_text(tokenizer):
     def func(dataset):
-        tokenized = tokenizer(dataset["text"], return_attention_mask=False, return_tensors="np")
+        tokenized = tokenizer(dataset["text"], return_attention_mask=False, return_tensors="np", padding=True)
```
[the remark this answers is not in the document]
Comment (payoto) [reply gptj-text-generation/data/mnli_data.py:83]: https://graphcore.slack.com/archives/C02969A3Z52/p1681487376639119

**Comment by vvvm23** [comment]
I think adding padding will change the behaviour as before the inputs got truncated. If the accuracy is similiar it is probably fine.

Did paperspace update their numpy version? Testing some older versions out and they support inhomogeneous arrays, but newer ones don't be default, which I think is where this error comes from.

**Comment by payoto** [comment]
Test has passed 82% accuracy
